### PR TITLE
Issue #40 Created CarDamageEntity class and DamageStatus enum

### LINKED ISSUE #40: Create CarDamageEntity

### PR DESCRIPTION
**Pull request by ChirkinaTanya**

### CODE REVIEW

**Review by nikaalin — APPROVED** [review]

**Review by a-deriabin — APPROVED** [review]

File: src/main/java/com/ifmo/epampractice/entities/CarDamageEntity.java [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+public class CarDamageEntity {
+    private int id;
+    private int orderId;
+    private String description;
+    private LocalDate accidentDate;
+    private int severity;
+    private int repairCost;
+    private DamageStatus status;
+
+    public CarDamageEntity(int id, int orderId, String description, LocalDate accidentDate, int severity, int repairCost, DamageStatus status) {
```
Comment (JustRevolt) [inline src/main/java/com/ifmo/epampractice/entities/CarDamageEntity.java]: Line is too long. Split it.

File: src/main/java/com/ifmo/epampractice/entities/CarDamageEntity.java [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+    }
+
+    public void setStatus(DamageStatus status) {
+        this.status = status;
+    }
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) return true;
+        if (o == null || getClass() != o.getClass()) return false;
```
Comment (JustRevolt) [inline src/main/java/com/ifmo/epampractice/entities/CarDamageEntity.java]: Add braces {}

File: src/main/java/com/ifmo/epampractice/entities/CarDamageEntity.java [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+        CarDamageEntity that = (CarDamageEntity) o;
+        return id == that.id &&
+                orderId == that.orderId &&
+                severity == that.severity &&
+                repairCost == that.repairCost &&
+                description.equals(that.description) &&
+                accidentDate.equals(that.accidentDate) &&
+                status == that.status;
```
Comment (JustRevolt) [inline src/main/java/com/ifmo/epampractice/entities/CarDamageEntity.java]: Compare by `Objects.equals()` 
This method helps to avoid null objects

**Review by JustRevolt — CHANGES_REQUESTED** [review]

File: src/main/java/com/ifmo/epampractice/entities/CarDamageEntity.java [hunk 4]
```diff
@@ -0,0 +1,101 @@
+package com.ifmo.epampractice.entities;
```
Comment (a-deriabin) [inline src/main/java/com/ifmo/epampractice/entities/CarDamageEntity.java]: change `entities` to `entity`

File: src/main/java/com/ifmo/epampractice/entities/CarDamageEntity.java (hunk 3 above)
Comment (a-deriabin) [reply src/main/java/com/ifmo/epampractice/entities/CarDamageEntity.java]: P.S. only use Objects.equals() for nullable fields (e.g. do not compare integers with it)

File: src/main/java/com/ifmo/epampractice/entities/CarDamageEntity.java (hunk 1 above)
Comment (ChirkinaTanya) [reply src/main/java/com/ifmo/epampractice/entities/CarDamageEntity.java]: fixed

File: src/main/java/com/ifmo/epampractice/entities/CarDamageEntity.java (hunk 2 above)
Comment (ChirkinaTanya) [reply src/main/java/com/ifmo/epampractice/entities/CarDamageEntity.java]: fixed

File: src/main/java/com/ifmo/epampractice/entities/CarDamageEntity.java (hunk 3 above)
Comment (ChirkinaTanya) [reply src/main/java/com/ifmo/epampractice/entities/CarDamageEntity.java]: fixed

File: src/main/java/com/ifmo/epampractice/entities/CarDamageEntity.java (hunk 4 above)
Comment (ChirkinaTanya) [reply src/main/java/com/ifmo/epampractice/entities/CarDamageEntity.java]: fixed

**Review by JustRevolt — APPROVED** [review]

**Review by PavelEzhkov — APPROVED** [review]

**Pull request merged**